### DISCUSSION TITLE
⭐️💬 Request for feedback: Would you like Discussions to be sorted by recent activity or newest post? ✨

### DISCUSSION THREAD

**Discussion by jayceeday** [community/community#31599] [Other Feature Feedback, Questions, & Ideas]
Hi everyone! I'm Jaycee, a Designer working on Discussions at GitHub.

Currently we sort Discussions by most **recent activity** on a post (ie. latest updated like new comments). 
Would you prefer it to be by default sorted by **newest post** (most recently created Discussion)? 

We would have the option to sort by both, but curious what you want for the default when you go to a discussion!
<img width="197" alt="Screenshot 2022-09-01 at 13 13 51" src="https://user-images.githubusercontent.com/86061938/187900918-d8330481-12f0-4106-bf58-dc1641ccfb50.png">

Thanks for your input 🤗

**Comment by DenverCoder1** [comment]
I often go to discussions to find unanswered questions to answer, and old posts that are getting recent activity are usually not the type that need answers, so those just get in the way of finding the newly posted, unanswered questions that I'm looking for.

Just adding my thoughts on why I voted for newest discussion post to be the default.

There's an argument for recent activity as well, which is simply because it's consistent with the current design, but I would not be using that sort mode as frequently.

**Reply by jayceeday** [reply]
Thank you for the feedback!

**Comment by jrbarnes9** [comment]
I think there's a third option here. "Recent activity" isn't very useful to us, as "activity" includes replies to comments. We'd like to be able to sort by the discussions that have had just new comments, not the ones that have had replies from our team.

**Reply by jayceeday** [reply]
Thank you for the feedback! This is out of scope for now but we'll take this into consideration in the future!
